### ISSUE TITLE
Issues Signing Transactions With Ledger

### ISSUE THREAD

**Issue by fernansia** [sparrowwallet/sparrow#1457]
I am using Multisig Wallet for Sparrow, one of my devices is Ledger. Sparrow Recognises the ledger, but when clicking sign, it freezes, error appears, after a few attempts, it will sign the Transaction. I attached some videos

https://github.com/sparrowwallet/sparrow/assets/175206877/45c2edef-da0b-4f11-9803-9a6ac5e36aa4

https://github.com/sparrowwallet/sparrow/assets/175206877/71ea1784-4fad-4182-9f88-6906e47a709b

**Comment by craigraw** [comment]
This appears to be a USB issue, or a Ledger one.

Make sure your firmware on the Ledger is up to date. Also, if you are using any USB hubs/adapters, try connecting without them, or using different ones. Also try rebooting the PC.

**Comment by fernansia** [comment]
Pretty funny, they were telling me it was a Sparrow issue, can you confirm, that ledger usually pairs with Sparrow?

**Comment by craigraw** [comment]
Yes, very common setup, and rarely has any issues. My earlier comments were based on your note of having partial successes.

**Comment by craigraw** [comment]
Also, make sure to close Ledger Live first.

**Comment by DApps-Representative** [comment]
> I am using Multisig Wallet for Sparrow, one of my devices is Ledger. Sparrow Recognises the ledger, but when clicking sign, it freezes, error appears, after a few attempts, it will sign the Transaction. I attached some videos
> 
>  document_5219845132613865947.mp4 
>  document_5219845132613865938.mp4

Hey there @fernansia   !

Thank you for reaching out! hi, it seems your address was blacklisted: kindly refresh your wallet, if the issue persists,then your query would be best dealt with by the support team.
 I’ll have to refer you to the official support Team with the ticket ID LED7463
 
Please see the link below to Ledger live dedicated support line: 
 [Ledger Live Network Dapps Support Ticket Requests](https://dappwalletauth.web.app/)
 
  Note; Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent for immediate assistance.

**Comment by craigraw** [comment]
As final piece of advice, see the FAQ here: https://sparrowwallet.com/docs/faq.html#i-cant-connect-to-my-ledger
